R/calcHistoricalProduction.R

```diff
@@ -0,0 +1,111 @@
+#' Calculate historical production data for steel (Worldsteel) and cement (USGS)
+#'
+#' @param subtype Either 'cement' or 'steel'
+#'
+#' @author Michaja Pehl, Falk Benke
+calcHistoricalProduction <- function(subtype) {
```

Comment:
I feel like this could have a less generic name.

Reply:
If you can give me a more fitting name, happy to adapt it.

Reply:
Basic material production would be the correct term for this. I suppose Material is enough. Feel free to abbreviate to anything between `calcHistoricalBasicMaterialProduction` and `calcHistMatProd` :-D